src/pages/guides/getting-started/concepts.mdx

```diff
@@ -2,96 +2,119 @@ export const description = 'Initial concepts of the Nitric framework'
 
 # Concepts
 
-Nitric is a framework for rapidly building cloud and serverless applications. It helps you define your apps in terms of the resources they need, then write the code for APIs, event subscribers and scheduled jobs. Unlike other cloud or container frameworks, apps built with Nitric can be deployed to AWS, Azure or Google Cloud all from the same code base.
+Nitric is a multi-language framework for rapidly building cloud applications that run on many different cloud providers such as AWS, Google Cloud or Microsoft Azure, with reduced boilerplate and infrastructure automation. This is often called **Infrastructure-_from_-Code**, since the core of your application becomes the source of true for both the application logic and deployment architecture.
```

Comment:
```suggestion
Nitric is a multi-language framework for rapidly building cloud applications that run on many different cloud providers such as AWS, Google Cloud or Microsoft Azure, with reduced boilerplate and infrastructure automation. This is often called **Infrastructure-_from_-Code** (IfC), since the core of your application becomes the source of truth for both the application logic and deployment architecture.
```

Reply:
Add the (IfC) as the abbreviation is used later